### PR TITLE
Wasm update for parside wasm: add missing Deref's

### PR DESCRIPTION
**Pull request by DmitryKuzmenko**
This update affects the only wasm part of cesride:
- Add `Deref` to each wrapper class to allow parside code to access the wrapped class. This is needed to create/re-wrap original rust classes in groups.

### CODE REVIEW

File: wasm/src/primitives/bexter.rs [hunk 1]
```diff
@@ -91,3 +91,11 @@ impl Deref for BexterWrapper {
         &self.0
     }
 }
+
+impl Wrap<Bexter> for BexterWrapper {
+    type Wrapper = BexterWrapper;
+
+    fn wrap(verfer: &Bexter) -> Self::Wrapper {
```
Comment (jasoncolburne) [inline wasm/src/primitives/bexter.rs:98]: Why not name all these `verfer` variables `matter` or explicitly the correct name?

**Review by jasoncolburne — COMMENTED** [review]
This is close. In the future it would be great if the demos you made in the other PR were actually exercised in CI, I have a pather PR up and this PR (#140) makes me realize that during the Bexter refactor I'd break `wasm`. But, tests are passing, giving me false confidence.

File: wasm/src/primitives/bexter.rs (hunk 1 above)
Comment (DmitryKuzmenko) [reply wasm/src/primitives/bexter.rs:98]: Oops, just a mustype. I'll fix.

**Comment by DmitryKuzmenko** [comment]
> This is close. In the future it would be great if the demos you made in the other PR were actually exercised in CI, I have a pather PR up and this PR (#140) makes me realize that during the Bexter refactor I'd break `wasm`. But, tests are passing, giving me false confidence.

I agree. Someday I'll add basic tests for this.

**Review by m00sey — APPROVED** [review]

**Pull request merged**